### ISSUE TITLE
Deleting all orphaned jobs actually nukes ALL downloads

### ISSUE THREAD

**Issue by Lustyn** [sabnzbd/sabnzbd#1672]
Issue title mostly speaks for itself, but I had the bright idea of clicking "delete all orphaned jobs" after a single orphaned job occuring, and upon doing this SABnzbd decided to, without confirmation, nuke every single folder in my downloads.

Bit unfortunate, but thankfully I didn't lose any important data.

**Comment by Safihre** [comment]
What SABnzbd version and OS are you on?
Do you maybe have extra logging enabled? You can see this in the Status and Interface settings window, if it's set to +Info for example.

**Comment by Lustyn** [comment]
I'm on 3.1.0 [23f86e9], with +Info logging. I was unable to find any indication of sabnzbd deleting things in the /logs/ directory but perhaps I was grepping for the wrong things.

**Comment by Safihre** [comment]
If you could maybe zip all the logs in the admin / logs directory and send them to me at safihre@sabnzbd.org? 
Maybe there's something in there.

**Comment by Lustyn** [comment]
I've sent the logs, and I grepped again and noticed there were some suspicious bits around the time the orphaned job was removed but nothing too indicative of what happened.

**Comment by Lustyn** [comment]
So I appear to have solved this issue, or rather, discovered what was causing my confusion.

Some background:
I have the typical setup with Radarr, Sonarr, etc. hooked into SABnzbd and prefer to keep the source files of all the things I download to ensure that if, for some reason, the original uploads die I have access to the untouched copies rather than the renamed/reorganized versions in my library.

So, I have been offloading data to a personal server and was double checking my upload and noticed it was about half of what I expected (since I use twice the typical storage due to storing two copies). I checked my downloads folder and a LARGE number of them were missing, almost all of them.

I was highly suspicious this could have been some combination of multiple pieces of software malfunctioning and erroneously deleting things since I use such a weird filesystem setup and had been experiencing some crashing and erroring on my entire setup.

Turns out, this was just me misunderstanding the behavior of Sonarr/Radarr. I had previously been using a torrent client instead of usenet which causes Radarr and Sonarr to do copies since people typically seed the content they download (like good citizens should). Well, when I switched to usenet Sonarr and Radarr silently started moving files instead of copying, resulting in a rather empty downloads folder containing only things I manually imported into Sonarr/Radarr by copy.

Phew, glad I figured that out.
